observatory/landing/form.js

```diff
@@ -0,0 +1,70 @@
+import { LitElement, css, html } from "lit";
+
+export class FormProgress extends LitElement {
+  static styles = css``;
+
+  static properties = {
+    _queryRunning: { state: true, type: Boolean },
+    hostname: { type: String },
+  };
+
+  constructor() {
+    super();
+    this._queryRunning = false;
+    this.hostname = "";
+  }
+
+  /**
+   * @param {Event} event
+   */
+  async _handleSubmit(event) {
+    event.preventDefault();
+    /** @type {HTMLInputElement | null | undefined} */
+    const input = this.shadowRoot?.querySelector("#host");
```

Comment:
Could use a `ref` here if you like: https://lit.dev/docs/templates/directives/#ref